proposals/0333-handle-scenario-where-no-valid-cert-is-available.md

```diff
@@ -34,8 +34,26 @@ If a PTU is completed and Core receives a valid certificate before the `Handshak
 
 ![handleScenarioWhereNoValidCertIsAvailable](https://user-images.githubusercontent.com/12716076/117061498-4a15c000-acf0-11eb-9907-ebc733d236ba.png)
 
+### Changes to OnPermissionsChange
+In order to notify an application when a valid certificate is found, changes are proposed to the `OnPermissionChange` notification:
+
+```
+    <function name="OnPermissionsChange" functionID="OnPermissionsChangeID" messagetype="notification" since="2.0">
+        <description>Provides update to app of which policy-table-enabled functions are available</description>
+        <param name="permissionItem" type="PermissionItem" minsize="0" maxsize="500" array="true" mandatory="true">
+            <description>Change in permissions for a given set of RPCs</description>
+        </param>
+        <param name="requireEncryption" type="Boolean" mandatory="false" since="6.0"/>
++       <param name="encryptionReady" type="Boolean" mandatory="false" since="x.x">
++           <description>If true, encryption is ready. If false, encryption is not ready. If omitted, mobile assumes encryption ready.</description>
++       </param>
+    </function>
+```
+
+In the case an app's StartService was NAK'd because Core did not have a valid certificate, when the app receives `OnPermissionChange` with `encryptionReady = true` it will know that it may retry it's StartService.
```

Comment:
```suggestion
In the case an app's StartService was NAK'd because Core did not have a valid certificate, when the app receives `OnPermissionsChange` with `encryptionReady = true` it will know that it may retry its StartService.
```